app/models/post.rb

```diff
@@ -0,0 +1,13 @@
+class Post < ApplicationRecord
+  belongs_to :user
```

Comment:
- [ ] Kindy add the foreign key and the class associated with the key eg if its a book belonging to a writer it would be `

```suggestion
  belongs_to :author_id, class_name: 'User'
```

Reply:
Well resolved 👍🏽

Reply:
Well resolved 👍🏽